code/modules/mob/living/carbon/human/monkey/monkey.dm

```diff
@@ -34,6 +34,18 @@
 
 GLOBAL_DATUM(the_one_and_only_punpun, /mob/living/carbon/human/species/monkey/punpun)
 
+/obj/item/clothing/under/punpun
+	name = "fancy uniform"
+	desc = "It looks like it was tailored for a monkey."
+	icon = 'icons/obj/clothing/under/civilian.dmi'
+	worn_icon = MONKEY_UNIFORM_FILE
+	icon_state = "punpun"
+	can_adjust = FALSE
+	female_sprite_flags = NO_FEMALE_UNIFORM
+	species_exception = list(/datum/species/monkey)
+	bodytypes_whitelist = BODYTYPE_MONKEY
```

Comment:
Do we need to use this at all? Digitigrade clothings (and now monkey clothings) change their icon already when they are equipped to a person with that bodytype. Couldn't this just be added to the waiter's suit as the monkey variation of it??

Reply:
Made it use the human waiter's suit instead

Reply:
does that mean we can remove this 
![image](https://user-images.githubusercontent.com/46540570/179638838-b5a0d60c-cd9c-4382-b7e1-94177cba349e.png)

Reply:
@Twaticus good catch will delete